src/lights/PointLightShadow.js

```diff
@@ -64,7 +67,7 @@ class PointLightShadow extends LightShadow {
 		const camera = this.camera;
 		const shadowMatrix = this.matrix;
 
-		const far = light.distance || camera.far;
+		const far = light.distance != null ? light.distance : DEF_DISTANCE;
```

Comment:
TBH, I don't understand this change.  `null` is not a valid value for `PointLight.distance`.

Reply:
Sorry, something wrong in my implementation, I'll fixed it in next PR.

Reply:
It should be `light.distance != 0 ? light.distance : DEF_DISTANCE;` or `light.distance || DEF_DISTANCE`. The behavior right now isn't correct I don't think.

Reply:
The new PullRequest: https://github.com/mrdoob/three.js/pull/22252